README.md

```diff
@@ -112,21 +109,21 @@ object Test extends App {
       println(s"Missing required field: '$fieldName'")
   }
 }
-
 ```
+
 See [unit tests](macros/src/test/scala/com/avast/cactus/CactusMacrosTest.scala) for more examples.
 
-Collections (`java.util.List<T>`) are converted to `scala.collection.immutable.Seq[T]`, concretely to `scala.collection.immutable.List[T]`. Default Scala
-`Seq[T]` is mutable and cannot be used in the target case class. Since both the GPB and case classes are meant to be immutable, this is intentional design of the converter. 
+Collections (`java.util.List<T>`) are converted to `scala.collection.immutable.Seq[T]` -
+concretely to `scala.collection.immutable.List[T]`. Default Scala `Seq[T]` is mutable and cannot be used in the target 
+case class. This is intentional design of the converter since both GPB and case classes are meant to be immutable. 
 
 ## Case class to GPB
 
-Case class can be converted to GPB the same easy way as in the other direction.  
+Case class can be mapped back to GPB as easily as in the other direction. The source case class can contain an arbitrary 
+scala collection (even mutable).
 
-The source case class can contain an arbitrary scala collection (even mutable).
-
-When mapping the collection, you have option to either add or not add `List` suffix to name of the case class field - see `fieldString2` vs. `fieldStringsList`
-in the example.
+You have the option to either add or not add `List` suffix to the name of the case class field - 
```

Comment:
This doesn't apply in general - only for collections, which is not mentioned in proposed version.